src/parse/asp/objects.go

```diff
@@ -967,6 +968,37 @@ func loadPluginConfig(state *core.BuildState, c pyDict) {
 	c[strings.ToUpper(pluginName)] = pluginNamespace
 }
 
+func toPyObject(key, val, toType string) pyObject {
+	if toType == "" || toType == "str" {
+		return pyString(val)
+	}
+
+	if val == "" {
+		return pyNone{}
+	}
+
+	if toType == "bool" {
+		if val == "true" || val == "True" {
```

Comment:
The config files support yes/no and on/off as well; should we include that here too?

Reply:
Good point. Done.